AssetRipper.Library/Ripper.cs

```diff
@@ -277,6 +278,9 @@ private void OverrideNormalExporters()
 			TextureAssetExporter textureExporter = new(Settings);
 			OverrideExporter<ITexture2D>(textureExporter); //Texture2D and Cubemap
 			OverrideExporter<ISprite>(textureExporter);
+			YamlSpriteExporter spriteExporter = new(Settings);
+			OverrideExporter<ISprite>(spriteExporter);
+			OverrideExporter<ISpriteAtlas>(spriteExporter);
```

Comment:
Use `ConditionalOverrideExporter` instead of a LibraryConfiguration constructor.